symbols/openstreetmap-carto/amenity/drinking_water.svg

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
This file should be deleted, isn't it?

Reply:
Well, it's not used, but it's the orginal file so I push it.